### ISSUE TITLE
Diagrams can't be read in dark mode

### ISSUE THREAD

**Issue by moorepants** [TJStienstra/brim#119]
Same issue as in SymPy:

![image](https://github.com/TJStienstra/brim/assets/276007/db16f52c-d5a1-477d-8359-c731893cbfca)

I'm using the lastest Firefox.

**Comment by TJStienstra** [comment]
Yeah, the problem is that this page is generated based on a jupyter notebook, so it is a bit more tricky to solve.
